CONTRIBUTING.md

```diff
@@ -1,120 +1,116 @@
 
-## 🛑 Important
+When you create a new snapshot test, run it with `pytest -vv tests/snapshot_tests/test_snapshots.py`.
+Because you just created this snapshot test, there is no history to compare against and the test will fail.
+After running the snapshot tests, you should see a link that opens an interface in your browser.
+This interface should show all failing snapshot tests and a side-by-side diff between what the app looked like when the test ran versus the historic snapshot.
 
-- Make sure to read the issue instructions carefully. If you are a newbie you should look out for some good first issues because they should be clear enough and sometimes even provide some hints. If something isn't clear, ask for clarification!
+Make sure your snapshot app looks like it is supposed to and that you didn't break any other snapshot tests.
+If everything looks fine, you can run `make test-snapshot-update` to update the snapshot history with your new snapshot.
+This will write to the file `tests/snapshot_tests/__snapshots__/test_snapshots.ambr`, which you should NOT modify by hand.
 
-- Add docstrings to all of your code (functions, methods, classes, ...). The codebase should have enough examples for you to copy from.
+([Read this](#makefile-commands) if the command `make` doesn't work for you.)
 
-- Write tests for your code.
+## Join the community
 
-- If you are fixing a bug, make sure to add regression tests that link to the original issue.
-  
-- If you are implementing a visual element, make sure to add snapshot tests. See below for more details.
+Seems a little overwhelming?
+Join our community on [Discord](https://discord.gg/uNRPEGCV) to get help!
 
-  
-### Snapshot Testing
-Snapshot tests ensure that things like widgets look like they are supposed to.
-PR [#1969](https://github.com/Textualize/textual/pull/1969) is a good example of what adding snapshot tests means: it amounts to a change in the file ```tests/snapshot_tests/test_snapshots.py```, that should run an app that you write and compare it against a historic snapshot of what that app should look like.
+## Makefile commands
 
-When you create a new snapshot test, run it with ```pytest -vv tests/snapshot_tests/test_snapshots.py.```
-Because you just created this snapshot test, there is no history to compare against and the test will fail automatically.
-After running the snapshot tests, you should see a link that opens an interface in your browser.
-This interface should show all failing snapshot tests and a side-by-side diff between what the app looked like when it ran VS the historic snapshot.
+Textual has a `Makefile` file that contains the most common commands used when developing Textual.
+([Read about Make and makefiles on Wikipedia.](https://en.wikipedia.org/wiki/Make_(software)))
+If you don't have Make, you can open the file `Makefile` with any text editor and read the rules yourself.
```

Comment:
I think we should focus on how to use Textual's Makefile. Its not a requirement that a developer understands the syntax of make, so we should omit the example of code.

Reply:
I'm with Will on this. I'd also add that, *if* it's easy to do, and without getting into a full tutorial in our own guide, because `make` isn't standard on Windows (like it is on Unix), we should consider looking for and pointing to a "how to get make on Windows" howto. But that's just a mild suggestion.

Reply:
Ok, see b12fa2ac7 for another suggestion.